owncloudApp/src/main/java/com/owncloud/android/presentation/spaces/SpacesListFragment.kt

```diff
@@ -214,10 +227,35 @@ class SpacesListFragment :
             }
         }
 
-        collectLatestLifecycleFlow(spacesListViewModel.menuOptions) { menuOptions ->
-            showSpaceMenuOptionsDialog(menuOptions)
+        collectLatestLifecycleFlow(spacesListViewModel.disableSpaceFlow) { event ->
+            event?.let {
+                when (val uiResult = event.peekContent()) {
+                    is UIResult.Success -> { showMessageInSnackbar(getString(R.string.disable_space_correctly)) }
+                    is UIResult.Loading -> { }
+                    is UIResult.Error -> { showErrorInSnackbar(R.string.disable_space_failed, uiResult.error) }
+                }
+            }
         }
 
+        collectLatestLifecycleFlow(spacesListViewModel.enableSpaceFlow) { event ->
+            event?.let {
+                when (val uiResult = event.peekContent()) {
+                    is UIResult.Success -> { showMessageInSnackbar(getString(R.string.enable_space_correctly)) }
+                    is UIResult.Loading -> { }
+                    is UIResult.Error -> { showErrorInSnackbar(R.string.enable_space_failed, uiResult.error) }
+                }
+            }
+        }
+
+        collectLatestLifecycleFlow(spacesListViewModel.deleteSpaceFlow) { event ->
+            event?.let {
+                when (val uiResult = event.peekContent()) {
+                    is UIResult.Success -> { showMessageInSnackbar(getString(R.string.delete_space_correctly)) }
+                    is UIResult.Loading -> { }
+                    is UIResult.Error -> { showErrorInSnackbar(R.string.delete_space_failed, uiResult.error) }
+                }
+            }
+        }
```

Comment:
That's ok in functional terms, but it seems like a code repetition for every operation (createSpaceFlow, editSpaceFlow, disableSpaceFlow, enableSpaceFlow, deleteSpaceFlow) where the only difference is the messages to show in the snackbar. Might it be refactored somehow?